ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-headers/gio-form-headers.component.html

```diff
@@ -30,8 +30,18 @@
     >
       <td class="gio-form-headers__table__header-row__td-key">
         <mat-form-field class="gio-form-headers__table__header-row__td-key__field" [floatLabel]="'never'">
-          <textarea formControlName="key" matInput placeholder="Name..." cdkTextareaAutosize cdkAutosizeMinRows="1"></textarea>
+          <textarea
+            formControlName="key"
+            matInput
+            placeholder="Name..."
+            cdkTextareaAutosize
+            cdkAutosizeMinRows="1"
+            [matAutocomplete]="headerNamesAutocomplete"
+          ></textarea>
         </mat-form-field>
+        <mat-autocomplete #headerNamesAutocomplete="matAutocomplete">
+          <mat-option *ngFor="let headerName of headerNames" [value]="headerName">{{ headerName }}</mat-option>
+        </mat-autocomplete>
```

Comment:
As a user I would expect the suggestion list to be filtered with what I'm typing in the field, what do you think?